package.json

```diff
@@ -79,7 +79,7 @@
   },
   "dependencies": {
     "@monaco-editor/react": "4.6.0",
-    "@taskany/colors": "1.1.0",
+    "@taskany/colors": "file:../colors",
```

Comment:
um, this isn't works im ci :)

Reply:
I'll wait until a new version of colors is released. Left it so as not to forget